src/mumble/ConnectedQuitBehavior.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_CONNECTEDQUITBEHAVIOR_H
+#define MUMBLE_CONNECTEDQUITBEHAVIOR_H
+
+#include <vector>
+
+
+/// This enum lists a series of quit options when connected to a server
+enum class ConnectedQuitBehaviorID {
+	ASK      = 0,
+	MINIMIZE = 1,
+	CLOSE    = 2
+};
+
+struct ConnectedQuitBehavior {
+	ConnectedQuitBehaviorID id;
+};
```

Comment:
What purpose does this separate struct have in this form?

Reply:
I just forgot to delete it ;)